package.json

```diff
@@ -2,12 +2,13 @@
   "name": "@resin/sbvr-types",
   "version": "1.2.1",
   "description": "SBVR type definitions.",
-  "main": "bin/types.js",
+  "main": "dist/SBVRTypes.node.min.js",
+  "browser": "dist/SBVRTypes.browser.min.js",
   "scripts": {
-    "lint": "resin-lint src test build.coffee",
+    "lint": "resin-lint src test",
     "pretest": "npm run lint && npm run prepublish",
     "test": "mocha",
-    "prepublish": "coffee build.coffee"
+    "prepublish": "webpack"
```

Comment:
Can you remove the webpack bundling please, and instead just compile the typescript to javascript - the reasoning for that is that if we bundle all of the modules here, then any child that wants to do a bundle themselves will be unable to deduplicate any of the modules eg if sbvr-types and the child use the same version of lodash and bluebird, then both will be bundled twice, once in sbvr-types and once in the child - adding up to a lot of increased size and hurting performance as there will be two copies of each function to be compiled/optimised.

To note this isn't saying to use the old bundling that was done, but to instead expect any user that wants the bundling to handle it via their method of choice, be that webpack, browserify, nodejs node_modules (ie no bundling), or whatever else